File: tasks/java/easy/GreetingApp.java
// Java - Easy
import java.util.Scanner;
public class GreetingApp {

    public static void main(String[] args) {
        Scanner kb = new Scanner(System.in);
        String name;

        System.out.print("Enter your name: ");
        name = kb.nextLine();
        
        System.out.println("Hello " + name);
    }
}

File: tasks/kotlin/easy/GreetingApp.kt
// Kotlin - Easy

fun main() {
    // TODO: Create a Simple Kotlin Program to Print User's Input
    print("Enter your name: ")
    val name = readLine()

    if (name.isNullOrBlank()) {
        println("Error: Name cannot be empty.")
    } else {
        println("Hello, $name!")
    }
}
